opentech/apply/activity/messaging.py

```diff
@@ -195,6 +239,12 @@ def send_message(self, message, user, submission, **kwargs):
         else:
             related_object = None
 
+        try:
+            # If this was a batch action we want to pull out the submission
+            submission = submissions[0]
+        except TypeError:
+            pass
+
```

Comment:
This makes sense after I looked at it for a minute but I think wrapping it in `if submissions` is clearer than a `try/except`
Not a deal breaker.